### PR TITLE
nuget 4.0 update

### PR DESCRIPTION
**Pull request by karann-msft**

### CODE REVIEW

File: content/PROD/Home.html [hunk 1]
```diff
@@ -31,7 +31,11 @@
 <section class="release">
     <h2>Latest NuGet Releases</h2>
     <p>
-        NuGet 3.5 for Visual Studio 2015 was released on October 27th, 2016. NuGet 2.12 for Visual Studio 2013 was released on June 27th 2016. <strong>Download</strong> these releases from http://nuget.org/downloads.
+        NuGet 4.0 for Visual Studio 2017 was release on November 17th, 2016. NuGet 3.5 for Visual Studio 2015 was released on October 27th, 2016. NuGet 2.12 for Visual Studio 2013 was released on June 27th 2016. <strong>Download</strong> these releases from http://nuget.org/downloads.
```
Comment (shishirx34) [inline content/PROD/Home.html]: was __released__ on ...

**Comment by skofman1** [comment]
@karann-msft , can you please send a PR against dev? We never merge directly to master. Thanks! 🌞

**Comment by 304NotModified** [comment]
You could change this PR so it points to dev. ;)

**Comment by skofman1** [comment]
Thanks @304NotModified ! Didn't know it was possible!
@karann-msft , should I merge? Is this PR still relevant?

File: content/PROD/Home.html (hunk 1 above)
Comment (karann-msft) [inline content/PROD/Home.html]: fixed.

**Comment by karann-msft** [comment]
The PR is relevant. Waiting for the exe to be available on the dist.

**Comment by 304NotModified** [comment]
It's just a change in the HTML file. This change should be really easy to review and merge.

**Comment by joelverhagen** [comment]
This information no longer exists on the front page.

**Comment by 304NotModified** [comment]
congrats, within a year!

**Pull request closed without merging**